frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts

```diff
@@ -230,12 +232,14 @@ export const gitPage = {
     }
   },
   verifyValidatedMessage: (gitUrl = 'https://github.com/sclorg/nodejs-ex.git') => {
-    cy.get(gitPO.gitSection.validatedMessage).should('not.have.text', 'Validating...');
+    cy.get(gitPO.gitSection.validatedMessage)
+      .should('not.include.text', 'Validating...')
+      .and('not.include.text', 'Repository URL to build and deploy your code from source');
```

Comment:
Can you please add this text to [static-text ](https://github.com/openshift/console/tree/master/frontend/packages/dev-console/integration-tests/support/constants/staticText)folder

Reply:
Added